templates/x86/test/units/mods.conf

```diff
@@ -127,10 +127,11 @@ configuration conf {
 
 	@Runlevel(1) include embox.compat.libc.test.math_test
 	@Runlevel(1) include embox.compat.libc.test.math_test_fp
+	@Runlevel(1) include embox.compat.libc.test.math_test_atan
 	@Runlevel(1) include embox.compat.libc.test.log10_test
 	@Runlevel(1) include embox.compat.libc.test.fmod_test
 	@Runlevel(1) include embox.compat.libc.test.sqrt_test
-    @Runlevel(1) include embox.compat.libc.test.exp_test
+  @Runlevel(1) include embox.compat.libc.test.exp_test
```

Comment:
Is it required

Reply:
The line was misaligned in my editor. It seems that my setting for the tab is different from the codestyle for .conf files.